File: Selenium/Activities/Activity11.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;

public class Activity11 {
    public static void main(String[] args) {
        WebDriver driver=new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println("Home page title is :"+driver.getTitle());
        //Select the checkbox
        WebElement checkBox=driver.findElement(By.name("toggled"));
        checkBox.click();
        System.out.println("checkbox selected status is: "+checkBox.isSelected());
        //Deselct the check box
        checkBox.click();
        System.out.println("checkbox selected status is: "+checkBox.isSelected());
        driver.quit();
    }
}

File: Selenium/Activities/Activity11.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Chrome() as driver:
    # Navigate to the URL
    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("Page title is: ", driver.title)

    checkbox = driver.find_element(By.CSS_SELECTOR, "input.willDisappear")
    # Click the checkbox to select it
    checkbox.click()
    # Verify if the checkbox is selected or not
    print("Checkbox is selected: ", checkbox.is_selected())
    # Click the checkbox again to uncheck it
    checkbox.click()
    # Verify if the checkbox is selected or not
    print("Checkbox is selected: ", checkbox.is_selected())